tests/test_api_jws.py

```diff
@@ -215,6 +215,27 @@ def test_decodes_valid_jws(self, jws, payload):
 
         assert decoded_payload == payload
 
+    def test_decodes_complete_valid_jws(self, jws, payload):
```

Comment:
If you start enforcing Mypy on CI, I'd suggest adding some annotations to the tests (and adding tests/ to the Mypy run), as they can help catch API mistakes, as this is the only place in the code that actually uses much of the library. We've done this recently on aiohttp-jinja2, where some annotations were incorrect and only getting noticed by users previously.

Reply:
Certainly would be useful, but it seems a much bigger issue to tackle and so I think is outside the scope for this particular PR. Help there would be welcome.

P.S. I'm not the maintainer of this project, just a contributor.

Reply:
Of course, just giving some ideas for the future.

If you want to tackle it, I'd start by adding a `.mypy.ini` file, for example:
https://github.com/aio-libs/aiohttp-jinja2/blob/master/.mypy.ini

That may be a little too strict for this project, so play around with the options (but, you'll want to change `disallow_untyped_defs` to `True` under tests, we skip it because we don't have any important things in the parameters).

Then, adding CI support can be done with something as simple as:
https://github.com/mlowijs/tesla_api/blob/typing/.github/workflows/ci.yaml#L16-L23

Let me know if you need any other help.